src/k-provider/ovp/services/base-entry-service.js

```diff
@@ -35,10 +35,11 @@ export default class OVPBaseEntryService extends OVPService {
    * @param {string} serviceUrl The base URL
    * @param {string} ks The ks
    * @param {string} entryId The entry ID
+   * @param {boolean} redirectFromEntryId whether the live entry should continue and play the VOD one after the live stream ends.
    * @returns {RequestBuilder} The request builder
    * @static
    */
-  static list(serviceUrl: string, ks: string, entryId: string): RequestBuilder {
+  static list(serviceUrl: string, ks: string, entryId: string, redirectFromEntryId: boolean): RequestBuilder {
```

Comment:
just a note: this filter option is the only one we have used for a "decade" so I don't foresee additional ones.
If we will have more options this will become a filter object